Homework/Hang_Nguyen/lesson_1/sort-array-by-parity.py

```diff
@@ -0,0 +1,23 @@
+class Solution:
+    def sortArrayByParity(self, nums):
+        last = len(nums) - 1
+        begin = 0
+        while begin < last: 
+            if nums[begin] % 2 != 0: 
+                temp = nums[begin]
```

Comment:
iirc, swap in python can be done by a,b = b,a

Reply:
Hi thay Quang,

I have updated as your feedback.

Many thanks,
Hang Nguyen